### PR TITLE
add travis initial setting

### PR DESCRIPTION
**Pull request by juliexue**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.5-dev" # 3.5 development branch
```
Comment (Nosajool) [inline .travis.yml]: Does this mean it'll run tests 7 times for each version of python?

Comment (juliexue) [reply .travis.yml]: yes it does, I'm just testing if it works - which version are we using?

Comment (Nosajool) [reply .travis.yml]: @davidozhang ^^

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  - "nightly" # currently points to 3.6-dev
+# command to install dependencies
+install: "pip install -r requirements.txt"
+# command to run tests
+script: nosetests
```
Comment (davidozhang) [inline .travis.yml]: new line

Comment (jerryzxliu) [reply .travis.yml]: @juliexue `"ensure_newline_at_eof_on_save": true` add this to your sublime setting

Comment (juliexue) [reply .travis.yml]: okay

**Review by davidozhang — APPROVED** [review]

**Pull request merged**